setup.py

```diff
@@ -89,8 +89,8 @@ def run(self):
             'pytest-pep8',
             'pytest-cov',
             'pyfakefs',
-            'sphinx<2',
-            'sphinx-rtd-theme<0.5',
+            'sphinx',
+            'sphinx-rtd-theme',
```

Comment:
Maybe update the maximum version restriction here as well? `"tzlocal>=2,<3", "google-auth-oauthlib>=0.4,<0.5",`

Reply:
Is there a reason for that? Do you need some specific versions?

Reply:
First of all, I would like to work with the latest versions of packages. Nothing should break within the major version. For example, `google-auth-oauthlib==0.5.0` works fine for me. Why not.

Reply:
I've updated the versions of `tzlocal` and `google-auth-oauthlib` to the newest.
But I'm keeping the major version cap because of possible incompatibility in future versions of the packages.

Reply:
Well, that broke some tests :)